job-runner/chain/redis_repo.go

```diff
@@ -0,0 +1,185 @@
+// Copyright 2017, Square, Inc.
+
+package chain
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/garyburd/redigo/redis"
+)
+
+// ChainKey is the keyspace for serialized Chains
+const CHAIN_KEY = "ChainById"
+
+var (
```

Comment:
These vars should be in `repo.go`, i.e. one source of truth wrt repo errors so the caller (whatever code is using the repo) can remain oblivious to the implementation (Redis repo, memory repo, etc. -- doesn't matter, they all return the same set of errors).